app/src/util/helpers.ts

```diff
@@ -254,33 +254,33 @@ export const getInputMethodology = (methodologyId: string) => {
 };
 
 export function convertKgToTonnes(
-  valueInTonnes: number | bigint,
+  valueInKg: number | bigint,
```

Comment:
Let's make sure we're not introducing a breaking change here, since this function is probably used in other places as well?

Reply:
thanks! I rolled back the change